website/docs/en/plugins/rspack/lightning-css-minimizer-rspack-plugin.mdx

```diff
@@ -17,19 +17,26 @@ module.exports = {
 
 ## Options
 
-### errorRecovery
+### include
 
-- **Type:** `boolean`
-- **Default:** `true`
+- **Type:** `string | RegExp | (string | RegExp)[]`
+- **Default:** `undefined`
 
-Whether to ignore invalid rules and declarations rather than erroring.
+Use this to specify which files should be minified.
 
-### unusedSymbols
+### exclude
 
-- **Type:** `string[]`
-- **Default:** `[]`
+- **Type:** `string | RegExp | (string | RegExp)[]`
+- **Default:** `undefined`
 
-A list of known unused symbols, including CSS class names, ids, and @keyframe names. The declarations of these will be removed.
+Use this to specify which files should be excluded from minification.
+
+### test
+
+- **Type:** `string | RegExp | (string | RegExp)[]`
+- **Default:** `undefined`
+
+Use this to provide a pattern that css files are matched against. If the filename matches the given pattern, it will be minified, otherwise it won't be.
```

Comment:
```suggestion
Use this to provide a pattern that CSS files are matched against. If the output filename matches the given pattern, it will be minified, otherwise it won't be.
```